<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>onepointwgt</class>
 <widget class="QWidget" name="onepointwgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>79</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QComboBox" name="cbType">
     <item>
      <property name="text">
       <string>PVO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>REB</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="cbEnable">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pbSettings">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../app/style.qrc">
       <normaloff>:/qss_icons/rc/undock.png</normaloff>:/qss_icons/rc/undock.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="4">
    <widget class="QPushButton" name="pbRm">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../app/style.qrc">
       <normaloff>:/qss_icons/rc/close.png</normaloff>:/qss_icons/rc/close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblPixmap">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../app/images.qrc">:/images/PvoAnt.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../app/images.qrc"/>
  <include location="../../app/style.qrc"/>
 </resources>
 <connections/>
</ui>
